CMake/curl-config.cmake.in

```diff
@@ -1,9 +1,15 @@
 @PACKAGE_INIT@
 
-if("@USE_OPENSSL@")
+if("@USE_OPENSSL@" OR "@CURL_ZLIB@" )
     include(CMakeFindDependencyMacro)
```

Comment:
I believe somebody will add another dependency shortly.
So why not include CMakeFindDependencyMacro unconditionally? 

```
include(CMakeFindDependencyMacro)
if("@USE_OPENSSL@")
  find_dependency(OpenSSL "@OPENSSL_VERSION_MAJOR@")
endif()
if("@USE_ZLIB@")
  find_dependency(ZLIB "@ZLIB_VERSION_MAJOR@")
endif()

```